app/cdap/components/FooterPanel/styles.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+
+export const useStyles = makeStyles({
+  containerProps: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-end',
+    alignItems: 'flex-start',
+    width: '100%',
+    position: 'absolute',
+    bottom: 54,
+  },
+  innnerContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'space-between',
+    backgroundColor: '#F3F6F9',
+    height: 40,
+    boxShadow: '0px -2px 2px #0000001a',
+    width: '100%',
+  },
+  zoomPercent: {
+    marginLeft: 2,
+    marginRight: 5,
+    marginTop: 8,
+  },
+  imgContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    padding: '8px 32px',
+    gap: '8px',
+    width: '88px',
+    height: '40px',
```

Comment:
use width 88 instead of 88px and height 40 instead of 40px

Reply:
done